### ISSUE TITLE
ParseError: 'import' and 'export' may appear only with 'sourceType: module

### ISSUE THREAD

**Issue by MateuszG** [urish/angular2-moment#28]
When is use amTimeAgo in template I have an error:
```

node_modules/angular2-moment/TimeAgoPipe.ts:3
import {Pipe, ChangeDetectorRef, PipeTransform, OnDestroy} from 'angular2/core';
^
ParseError: 'import' and 'export' may appear only with 'sourceType: module'

```
My package.js
```

{
  "dependencies": {
    "angular2": "2.0.0-beta.6",
    "angular2-moment": "^0.4.3",
    "es6-promise": "3.0.2",
    "es6-shim": "^0.33.13",
    "ionic-angular": "2.0.0-beta.3",
    "ionic-native": "^1.1.0",
    "ionicons": "3.0.0-alpha.3",
    "lodash": "^4.0.1",
    "moment": "^2.11.1",
    "reflect-metadata": "0.1.2",
    "rxjs": "5.0.0-beta.0",
    "zone.js": "0.5.14"
  },
  "devDependencies": {
    "del": "2.2.0",
    "gulp": "3.9.1",
    "gulp-watch": "4.3.5",
    "ionic-gulp-browserify-typescript": "^1.0.0",
    "ionic-gulp-fonts-copy": "^1.0.0",
    "ionic-gulp-html-copy": "^1.0.0",
    "ionic-gulp-sass-build": "^1.0.0",
    "ionic-gulp-scripts-copy": "^1.0.0"
  },
  "cordovaPlugins": [
    "cordova-plugin-whitelist",
    "cordova-plugin-device",
    "cordova-plugin-console",
    "cordova-plugin-splashscreen",
    "ionic-plugin-keyboard",
    "cordova-plugin-statusbar"
  ],
  "cordovaPlatforms": [
    "ios",
    {
      "platform": "ios",
      "version": "",
      "locator": "ios"
    }
  ],
}
```

**Comment by urish** [comment]
Hi,

It seems like your setup does not parse typescript files inside node_modules. One thing you can try is explicitly import the javascript file:

    import {TimeAgoPipe} from 'angular2-moment/TimeAgoPipe.js'; 

See #11 for discussion

**Comment by bibitoo** [comment]
same issure on ionic2

**Comment by elineopsommer** [comment]
Here same issue too

**Comment by urish** [comment]
This should be fixed as part of angular2-moment 0.8.0:

    import {TimeAgoPipe} from 'angular2-moment';

Should do the trick.
